src/d3a/d3a_core/area_serializer.py

```diff
@@ -160,13 +160,3 @@ def optional(attr):
 def area_from_string(string, config=None):
     """Recover area from its json string representation"""
     return area_from_dict(json.loads(string), config)
-
```

Comment:
Better to keep this function and not to delete it. That way we might reuse it if we need it in the future.

Reply:
Sure Spyros, I have kept the method as it is and now Integration tests are passing.